authors/apps/articles/views.py

```diff
@@ -25,6 +26,9 @@ class ArticleNotFound(Exception):
 class RatingNotFound(Exception):
     pass
 
+class ReportNotFound(Exception):
+    pass
+
```

Comment:
Can't the above Exception classes be placed in a separate module?

Reply:
done